src/core/map.js

```diff
@@ -447,7 +458,7 @@ geo.map = function (arg) {
     m_rotation = rotation;
 
     var bounds = m_this.boundsFromZoomAndCenter(
-        m_zoom, m_center, m_rotation, null);
+        m_zoom, m_center, m_rotation, null, ignoreRotationFunc);
```

Comment:
During a transition, if we in discreteZoom mode and are animating both zoom and rotation, if we don't pass this flag we may end up discretizing the zoom level during the transition.